README.md

```diff
@@ -1 +1,47 @@
+# Portfolio
+Hi. I’m Islam, a Full-stack Developer I'm showing my experience of creating successful websites.
```

Comment:
- [ ] Please add the description of the project not a summary of who you are. e.g "Mobile version and setup of the figma design"